code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -538,6 +566,7 @@
 /datum/tarot/the_moon
 	name = "XVIII - The Moon"
 	desc = "May you find all you have lost."
+	extended_desc = "warps the user to a random place of intrest for miners and explorers, within the sector they're in first."
```

Comment:
```suggestion
	extended_desc = "teleports the user to a random place of interest for miners and explorers within the sector they're in first."
```

The end of the sentence is not clear for me, is this the sector where the user is in or the miners/explorers are in?